.pylintrc

```diff
@@ -92,7 +92,7 @@ evaluation=10.0 - ((float(5 * error + warning + refactor + convention) / stateme
 
 # Template used to display messages. This is a python new-style format string
 # used to format the message information. See doc for all details
-#msg-template=
+msg-template=[{msg_id}] {path}:{line:3d}:{column}: {msg}
```

Comment:
Will this mess up parsing the messages by other editors? (vscode, vim ...)

Reply:
It works well with VSCode. I don't think it will mess up parsing with other editors.